### DISCUSSION TITLE
How to ensure `check_suite` is only triggering on success?

### DISCUSSION THREAD

**Discussion by developher-net** [ridedott/merge-me-action#997] [Q&A]
How is the merge-me-action checking for successful completion when using `check_suite` as a trigger?

From [readme.md](https://github.com/ridedott/merge-me-action#public-repositories)
```yaml
name: Merge me!

on:
  check_suite:
    types:
      - completed
```

I'm having a hard time finding reliable information, but from my testing it appears that on a repo with github actions, `check_suite` is firing multiple times, and thus an automerge workflow is triggered multiple times. From the documentation, it doesn't seem that `completed` means successful completion, but again, [documentation is unclear here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#check_suite)

**Comment by deanhigh** [comment]
Hi, 

I agree it could be a little clearer, I am relatively new to more advanced github workflows and I found the webhook payloads doc helpful for this today. 

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#check_suite

I ended up with this which worked for my scenario : 
```
on:
  check_suite:
    conclusion: success
```
